lib/mongoid/contextual/mongo.rb

```diff
@@ -704,11 +773,15 @@ def acknowledged_write?
       # @param [ Object ] The value for the given field name
       def extract_value(attrs, field_name)
         def fetch_and_demongoize(d, meth, klass)
```

Comment:
Is this nested method definition required to be nested? https://stackoverflow.com/questions/39313535/ruby-doesnt-have-nested-methods-right-so-what-is-this-and-is-there-a-better

Doesn't look like it uses any variables not local to it, so it should be un-nested and gain a docstring?

Reply:
This is fixed in https://github.com/mongodb/mongoid/pull/5332